lib/fpm/package/deb.rb

```diff
@@ -181,6 +181,10 @@ class FPM::Package::Deb < FPM::Package
     next File.expand_path(file)
   end
 
+  option "--systemd-enable", :flag , "Enable service on install or upgrade", :default => true
+
+  option "--systemd-auto-start", :flag , "Start service after install or upgrade", :default => true
```

Comment:
I would prefer this default to false. Otherwise, we're breaking behavior from a previous release where services were not automatically started. Users wishing to have a service auto start should be required to use this flag.

Reply:
I've defaulted both to false